app/settings/account.tsx

```
@@ -11,84 +39,127 @@
+                        label="Username"
+                        value={formData.username}
+                        onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                            setFormData(prev => ({ ...prev, username: e.target.value }))}
+                        className="w-full"
+                        leadingIcon={null}
+                        trailingIcon={null}
+                    />
+                </div>
+
+                <div>
+                    <label className="block text-sm font-medium mb-2">Country</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)